packages/altair-app/src/app/modules/altair/pipes/sort/sort.pipe.ts

```diff
@@ -0,0 +1,21 @@
+import { Pipe, PipeTransform } from '@angular/core';
+
+@Pipe({
+  name: 'sort',
+})
+export class SortPipe implements PipeTransform {
+  transform(list: any[], key?: string): any {
```

Comment:
```suggestion
  transform(list: Record<string, any>[], key?: string): any {
```
if we intend to sort by `key` where `key` is a property of the list item, then maybe it makes sense to type this as `Record<string, any>` instead?